app/controllers/api/registrations_controller.rb

```diff
@@ -1,12 +1,20 @@
 class Api::RegistrationsController < DeviseTokenAuth::RegistrationsController
   def create
     customer = Stripe::Customer.list(email: params[:email]).data.first
-    customer ||= Stripe::Customer.create({ email: params[:email], source: params[:stripeToken] })
+    customer ||= Stripe::Customer.create({ email: params[:email]})
```

Comment:
What dose the 
`||=` really do?

Reply:
https://www.ruby-forum.com/t/the-definitive-list-of-or-equal-threads-and-pages/136446

Reply:
https://www.ruby-forum.com/t/the-definitive-list-of-or-equal-threads-and-pages/136446

Reply:
it is the same as typing unless customer in the end of the line, solargraph put pipes there. The idea is that if customer is undefined execute that line